{"name": "adding new address", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF61C2852A2+57122]\n\t(No symbol) [0x00007FF61C1FEA92]\n\t(No symbol) [0x00007FF61C0CE3AB]\n\t(No symbol) [0x00007FF61C107D3E]\n\t(No symbol) [0x00007FF61C107E2C]\n\t(No symbol) [0x00007FF61C140B67]\n\t(No symbol) [0x00007FF61C12701F]\n\t(No symbol) [0x00007FF61C13EB82]\n\t(No symbol) [0x00007FF61C126DB3]\n\t(No symbol) [0x00007FF61C0FD2B1]\n\t(No symbol) [0x00007FF61C0FE494]\n\tGetHandleVerifier [0x00007FF61C52EF82+2849794]\n\tGetHandleVerifier [0x00007FF61C581D24+3189156]\n\tGetHandleVerifier [0x00007FF61C57ACAF+3160367]\n\tGetHandleVerifier [0x00007FF61C316D06+653702]\n\t(No symbol) [0x00007FF61C20A208]\n\t(No symbol) [0x00007FF61C2062C4]\n\t(No symbol) [0x00007FF61C2063F6]\n\t(No symbol) [0x00007FF61C1F67A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\add_address.py\", line 13, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I logged into my account and got navigated to Homepage", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF61C2852A2+57122]\n\t(No symbol) [0x00007FF61C1FEA92]\n\t(No symbol) [0x00007FF61C0CE3AB]\n\t(No symbol) [0x00007FF61C107D3E]\n\t(No symbol) [0x00007FF61C107E2C]\n\t(No symbol) [0x00007FF61C140B67]\n\t(No symbol) [0x00007FF61C12701F]\n\t(No symbol) [0x00007FF61C13EB82]\n\t(No symbol) [0x00007FF61C126DB3]\n\t(No symbol) [0x00007FF61C0FD2B1]\n\t(No symbol) [0x00007FF61C0FE494]\n\tGetHandleVerifier [0x00007FF61C52EF82+2849794]\n\tGetHandleVerifier [0x00007FF61C581D24+3189156]\n\tGetHandleVerifier [0x00007FF61C57ACAF+3160367]\n\tGetHandleVerifier [0x00007FF61C316D06+653702]\n\t(No symbol) [0x00007FF61C20A208]\n\t(No symbol) [0x00007FF61C2062C4]\n\t(No symbol) [0x00007FF61C2063F6]\n\t(No symbol) [0x00007FF61C1F67A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\add_address.py\", line 13, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "3d58655c-ddee-4971-a4b3-bb15891ca279-attachment.png", "type": "image/png"}], "start": 1693459489643, "stop": 1693459489849}, {"name": "When I click on my account and go to address option", "status": "skipped", "start": 1693459492723, "stop": 1693459492731}, {"name": "And click on add a new address", "status": "skipped", "start": 1693459492731, "stop": 1693459492731}, {"name": "Then entered all the details and added it", "status": "skipped", "start": 1693459492731, "stop": 1693459492731}], "start": 1693459481297, "stop": 1693459492731, "uuid": "1f18639c-b7d6-48d7-9776-b43dac4b8118", "historyId": "12dd1a5d88f4de8a6e7a648477971cb2", "fullName": "print transactions history: adding new address", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "address"}, {"name": "feature", "value": "print transactions history"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}